### PR TITLE
Removed mix of import/from statements for the same library (pendulum)…

### PR DESCRIPTION
**Pull request by spyrostz**
…. Fixed ordering on imports on multiple test files.

### CODE REVIEW

**Review by faizan2590 — DISMISSED** [review]
LGTM

File: tests/test_area.py [hunk 1]
```diff
@@ -15,26 +15,27 @@
...
 from collections import OrderedDict
 from unittest.mock import MagicMock
-import unittest
+
+from d3a_interface.constants_limits import ConstSettings, GlobalConfig
 from parameterized import parameterized
+from pendulum import duration, today
+
+from d3a import constants
+from d3a.constants import TIME_ZONE
```
Comment (fievelk) [inline tests/test_area.py:27]: Based on the conversation we had, these two imports should be unified into one style, right?

File: tests/test_external_mixin.py [hunk 2]
```diff
@@ -1,20 +1,22 @@
+import json
 import unittest
 import uuid
-import json
 from unittest.mock import MagicMock
+
+from d3a_interface.constants_limits import ConstSettings
+from d3a_interface.constants_limits import GlobalConfig
```
Comment (fievelk) [inline tests/test_external_mixin.py:7]:
```suggestion
from d3a_interface.constants_limits import ConstSettings, GlobalConfig
```

File: tests/test_external_mixin.py [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
 from parameterized import parameterized
 from pendulum import now, Duration
+
+import d3a.models.strategy.external_strategies
+from d3a.constants import DATE_TIME_FORMAT
+from d3a.d3a_core.singletons import external_global_statistics
```
Comment (fievelk) [inline tests/test_external_mixin.py:13]: I have the feeling that these imports are not optimized. Can it be?

**Review by fievelk — APPROVED** [review]
Just two smaller comments but nothing major.

File: tests/test_external_mixin.py (hunk 3 above)
Comment (spyrostz) [reply tests/test_external_mixin.py:13]: Thanks, you were right they were not optimized. Will do that too, thanks again!

File: tests/test_area.py (hunk 1 above)
Comment (spyrostz) [reply tests/test_area.py:27]: Indeed, will fix that, thanks!

**Review by fievelk — APPROVED** [review]

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Pull request merged**